jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Modal.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.jdi.light.ui.bootstrap.elements.complex;
+
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.bootstrap.elements.common.Button;
+import com.epam.jdi.light.ui.bootstrap.elements.common.Text;
+
+public class Modal extends Section {
+    @UI(".modal-header .modal-title")
+    public Text title;
+    @UI(".modal-header button")
+    public Button closeX;
+}
```

Comment:
Some suggestions from @RomanIovlev:

```
public class Modal extends Section {
    @UI(".modal-header .modal-title")
    public Text title;

    public void close() {
        linked(".modal-header button", "Close").click();
    }
}
```

Reply:
Stop. Button closeX has to be removed also. Jst a moment

Reply:
button closeX removed